deploy/osd-rebalance-infra-nodes/06-osd-rebalance-infra-nodes-openshift-security.RoleBinding.yaml

```diff
@@ -0,0 +1,15 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: osd-rebalance-infra-nodes-openshift-security
+  namespace: openshift-security
+subjects:
+- kind: ServiceAccount
+  name: osd-rebalance-infra-nodes
+  namespace: openshift-monitoring
```

Comment:
Should this be `openshift-security`?

Reply:
Thank you for review. I place the SA in openshift-monitoring since we are dealing with 3 workloads, and two of them are in openshift-monitoring, and the cronjob is kind of monitoring the infra workloads. And bind the role permissions to the SA, including deleting pods in openshift-security, deleting pods/pvcs in openshift-monitoring and list nodes.

Reply:
Got it, thanks for clarifying!